api/src/main/java/net/jqwik/api/Arbitraries.java

```diff
@@ -421,6 +422,65 @@ public static StringArbitrary strings() {
 		return ArbitrariesFacade.implementation.strings();
 	}
 
+	/**
+	 * Create an arbitrary that generates valid E-Mail addresses.
+	 *
+	 * @return a new arbitrary instance
+	 */
+	public static int i = 0;
+	@API(status = EXPERIMENTAL, since = "1.38")
+	public static Arbitrary<String> emails() {
```

Comment:
Implementation of emails() looks good to me. Please move it to ArbitrariesFacadeImple